examples/templates/aws-linux/main.tf

```diff
@@ -239,17 +240,18 @@ Content-Transfer-Encoding: 7bit
 Content-Disposition: attachment; filename="userdata.txt"
 
 #!/bin/bash
-sudo -u ${local.linux_user} sh -c '${coder_agent.main.init_script}'
+sudo -u ${local.linux_user} sh -c '${coder_agent.dev[0].init_script}'
 --//--
 EOT
+  ) : ""
```

Comment:
Nice! TIL!

You can also use `try` so that Terraform provides a value even if something fails.

For example:

```hcl
  user_data = <<EOT
Content-Type: multipart/mixed; boundary="//"
MIME-Version: 1.0
@@ -239,17 +240,18 @@ Content-Transfer-Encoding: 7bit
Content-Disposition: attachment; filename="userdata.txt"
#!/bin/bash
sudo -u ${local.linux_user} sh -c '${try(coder_agent.dev[0].init_script, "")}'
--//--
EOT
```

Reply:
Nice. More cleaner

Reply:
@bpmct This works and is cleaner. Should we update the template?

Reply:
sure